Bottom: fb2021971529066688b51190af2eae46184a4a04
Top:    9714d100aa4e8dbc07250b021a078fdebfe48af2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:56:12 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 7039457..55571cf 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1251,8 +1251,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
